### DISCUSSION TITLE
Airflow sqlalchemy.exc.InvalidRequestError: Table 'ab_permission_view_role' is already defined for this MetaData instance. Specify 'extend_existing=True' to redefine options and columns on an existing Table object.

### DISCUSSION THREAD

**Discussion by naani10** [apache/airflow#31321] [Q&A]
Hi,

I am trying to install airflow 2.5.3 also 2.6.0 on AKS 1.25.

I am getting this error in sync user pod

sqlalchemy.exc.InvalidRequestError: Table 'ab_permission_view_role' is already defined for this MetaData instance. Specify 'extend_existing=True' to redefine options and columns on an existing Table object.

also i am getting error in airflow web pod

sqlalchemy.exc.InvalidRequestError: Multiple classes found for path "Permission" in the registry of this declarative base. Please use a fully module-qualified path.

KIndly post your suggestions
i am using below chart 
https://github.com/airflow-helm/charts

Thanks..

**Comment by potiuk** [comment]
I think you should open an issue there. We have a "wait-for-airlfow-mitgrations" init containers in our chart that prevent this from happening - not sure what is used in the chart that you mention.

**Comment by thesuperzapper** [comment]
@potiuk The issue was actually with @naani10's custom authentication module for flask, see https://github.com/airflow-helm/charts/issues/747#issuecomment-1570748218.

The issue is the same as [this user's problem](https://github.com/apache/airflow/issues/27138), where they were importing the `User` object from `flask_appbuilder.security.sqla.models`, but the correct import (since airflow 2.3.0) is from `airflow.www.fab_security.sqla.models`.

We might want to warn users about this in the docs somewhere because for many years airflow used the old location.

**Reply by potiuk** [reply]
Sure  Feel free to open PR about it. I think you know now best what exactly the problem is and we usually ask those who know best to contribute such clarifications usually those who investigate it also know best which docs to update . And it's almost as easy as commenting/creating issue - 'suggest a change on this page' and you will get PR opened where you can add such note/warning without really leaving GitHub UI.

**Reply by thesuperzapper** [reply]
I agree, the only place I can see for this sort of warning might be this page: https://airflow.apache.org/docs/apache-airflow/2.6.1/administration-and-deployment/security/webserver.html

But since this sort of thing is a pretty advanced use-case, I am not sure it's worth putting there, because it might confuse users who don't need this information.

It might just be best for people to run into this when it fails, lol.

**Reply by potiuk** [reply]
Probably you are right. GitHub issues for edge cases is  quite a good documentation
